packages/subgraph/test/helpers/helpers.ts

```diff
@@ -468,13 +468,13 @@ export const getOrder = (blockNumber?: number, logIndex?: number) => {
 export function calculateMaybeCriticalAtTimestamp(
     updatedAtTimestamp: string,
     balanceUntilUpdatedAt: string,
-    totalDeposit: string,
-    totalNetFlowRate: string
+    totalNetFlowRate: string,
 ) {
-    if (BigNumber.from(balanceUntilUpdatedAt).lte(BigNumber.from("0"))) return "0";
-    if (BigNumber.from(totalNetFlowRate).gte(BigNumber.from("0"))) return "0";
-    const criticalTimestamp = BigNumber.from(balanceUntilUpdatedAt).add(BigNumber.from(totalDeposit)).div(BigNumber.from(totalNetFlowRate).abs());
-    const calculatedCriticalTimestamp = criticalTimestamp.add(BigNumber.from(updatedAtTimestamp));
+    if (toBN(totalNetFlowRate).gte(toBN("0"))) return null;
+    if (toBN(balanceUntilUpdatedAt).eq(toBN("0"))) return "0";
+    if (toBN(balanceUntilUpdatedAt).lt(toBN("0"))) return updatedAtTimestamp;
```

Comment:
This implementation is not the same as the other one now.

Reply:
The only change in sdk-core would be to set the possibility that you will be returned null. Shouldn't be breaking to do this.

Reply:
Done